.docker/Dockerfile

```diff
@@ -42,5 +42,5 @@ ENV PATH "/app/shacl-${VERSION}/bin:${PATH}"
 COPY --from=jre-build /javaruntime $JAVA_HOME
 COPY --chmod=0755 --from=jre-build /app/shacl-${VERSION} /app/shacl-${VERSION}
 COPY --chmod=0755 --from=jre-build "/app/.docker/entrypoint.sh" "/entrypoint.sh"
-
+RUN sed -i 's/\r//g' entrypoint.sh
```

Comment:
If possible, I would prefer using a `.gitattributes` file to tell git that this specific file should have a linux file ending (\n).
This being a bash script, it's not meant to be executed in a windows environment anyways.
see e.g.:  https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=linux#per-repository-settings

Reply:
See also: https://stackoverflow.com/questions/71081340/prevent-vscode-to-crlf-files